src/interfaces/exportManager.ts

```diff
@@ -1,22 +1,20 @@
-import { FeatureCollection } from '@turf/turf';
+import { FeatureCollection } from "@turf/turf";
 import { CreateExportTaskRequest, CreateExportTaskResponse, TaskParameters } from './task';
 
 /* eslint-disable @typescript-eslint/naming-convention */
-export interface GetEstimatedSizeParams {
+export interface GetEstimationsRequest {
   ROI?: FeatureCollection;
   additional?: Record<string, unknown>;
 }
 
-export interface GetFileSizeParams {
-  filePath: string;
-  additional?: Record<string, unknown>;
+export interface GetEstimationsResponse {
+  estimatedFileSize: number;
```

Comment:
if result is unknown - should it be undefined or 0?

Reply:
should be undefined imo, fixed.